src/app/themes/theme.js

```diff
@@ -22,7 +24,13 @@ const theme = createMuiTheme({
       dark: '#c2c6ca',
       contrastText: '#000000',
     },
+    error: {
+      main: '#c44f65',
```

Comment:
Is this color used somewhere in this feature?

Reply:
its the color for the error inputs brought in on the Swagger Ticket due to the MUI default red not having enough contrast to be 508 compliant

Reply:
If it's a color needed for a different ticket, then it should be in that ticket